src/Demo.vue

```diff
@@ -66,8 +66,12 @@
 </template>
 
 <script>
+  import { A11yDialog } from '../dist/vue-a11y-dialog.esm'
```

Comment:
Noice 👍 

You may already be updating the `demo.js` but I'd test it. I don't think it'll work using default import without also doing the `A11yDialog.install(app)` bit (but my local branch is diverged from what you have here so have not verified that)